app/celery/scheduled_tasks.py

```diff
@@ -180,3 +181,12 @@ def switch_current_sms_provider_on_slow_delivery():
             )
 
             dao_toggle_sms_provider(current_provider.identifier)
+
+
+@notify_celery.task(name='timeout-job-statistics')
+@statsd(namespace="tasks")
+def timeout_job_statistics():
+    updated = dao_timeout_job_statistics(current_app.config.get('SENDING_NOTIFICATIONS_TIMEOUT_PERIOD'))
```

Comment:
Why assume that the status of the unaccounted for notifications are failed. Could we do a count of the status of the notifications here instead?